source/api/app.py

```diff
@@ -65,6 +65,29 @@ def list_datasets():
         logger.error(e)
         return {"Status": "Error", "Message": e}
 
+@app.route('/describe_dataset', cors=True, methods=['POST'], authorizer=authorizer)
+def describe_dataset():
+    """
+    Describe the schema and properties of an existing AMC dataset.
+
+    Returns:
+
+    .. code-block:: python
+
+        {"dataSets": [...]}
+    """
+    log_request_parameters()
+    try:
+        data_set_id = app.current_request.json_body['dataSetId']
```

Comment:
Can you add this new api to `test_api_integration.py`

Reply:
Thanks for the comment, added in newest commit